doc/lua_api.txt

```diff
@@ -2380,6 +2380,10 @@ Strings that need to be translated can contain several escapes, preceded by `@`.
       * nil: return all entries,
       * true: return only subdirectory names, or
       * false: return only file names.
+* `minetest.safe_file_write(path, content)`: returns boolean indicating success
+    * Replaces contents of file at path with new contents in a safe (atomic) way.
+      Use this instead of below code when writing e.g. database files:
+      `local f = io.open(..., "w"); f:write(newtext); f:close()`
```

Comment:
How about using the function parameters here for this example?
Also, it's the `"wb"` mode now.

i.e. something like this `local f = io.open(path, "wb"); f:write(content); f:close()` ?